### DISCUSSION TITLE
Link to webinar is broken

### DISCUSSION THREAD

**Discussion by julian202** [polarfire-soc/polarfire-soc-documentation#346] [General]
Link to Training#1: Introduction to the PolarFire® SoC Icicle Kit  is broken. The link to the seminar is found in this page: https://www.microchip.com/en-us/education/technical-learning-center/webinars/fpga-webinars/risc-v-innovation-unleashed

**Comment by hughbreslin** [comment]
Hi @julian202 where is the broken link so we can get it fixed?
